diff --git a/test-support/setup-middleware-reporter.d.ts b/test-support/setup-middleware-reporter.d.ts
index 947f3d5a5e92a4238f115a586bf552b34c97ab10..4360fcea65b4e6a1802ff60fb095c61d572ab413 100644
--- a/test-support/setup-middleware-reporter.d.ts
+++ b/test-support/setup-middleware-reporter.d.ts
@@ -1,4 +1,3 @@
-/// <reference types="ember__test-helpers" />
 import { currentRouteName } from '@ember/test-helpers';
 import { AxeResults, Result } from 'axe-core';
 export interface TestMetadata {
